OpenEdXMobile/src/main/java/org/edx/mobile/social/SocialProvider.java

```diff
@@ -36,7 +37,7 @@ public SocialError(Exception ex) {
     void getUser(Callback<SocialMember> callback);
 
     /**
-     *   Gets the user object for the current logged in user
+     * Gets the user object for the current logged in user
      */
-    void getUserInfo(Context context, SocialFactory.SOCIAL_SOURCE_TYPE socialType, String accessToken,  SocialLoginDelegate.SocialUserInfoCallback userInfoCallback);
+    void getUserInfo(Context context, SocialSourceType socialType, String accessToken, SocialLoginDelegate.SocialUserInfoCallback userInfoCallback);
```

Comment:
Not converting it to Kotlin right now. We need to check whether the fields are nullable or not and then proceed to test the Social Logins. It will went out of scope.